packages/core/src/components/alert/alert.tsx

```diff
@@ -8,11 +8,21 @@ import clsx from 'clsx';
 })
 export class Alert {
   @Prop()
-  level: 'success' | 'info' | 'warning' | 'danger';
+  status!: 'success' | 'info' | 'warning' | 'danger';
+
+  statusAlt = {
+    success: 'Gelukt:',
+    info: 'Opmerking:',
+    warning: 'Waarschuwing:',
+    danger: 'Fout:'
+  };
 
   render() {
     return (
-      <div class={clsx('alert', { [`alert-${this.level}`]: this.level })} role="alert">
+      <div class={clsx('alert', { [`alert-${this.status}`]: this.status })} role="alert">
```

Comment:
Als `status` niet meer optioneel is, dan hoeft hier ook geen check meer worden gedaan..

Reply:
Goed punt, de check is weg